presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -166,4 +176,22 @@ private void initCatalogAndSchema()
             schema = parts.get(1);
         }
     }
+
+    private static Properties getMergedProperties(URI uri, Properties driverProperties)
+            throws SQLException
+    {
+        Properties result = new Properties();
+        result.putAll(ConnectionProperties.getDefaults());
+        result.putAll(driverProperties);
```

Comment:
I'm not sure if you can call this properties as `driverProperties`. I would rather leave it as just `properties`.